sys/src/9/port/syscallfmt.c

```diff
@@ -264,20 +264,25 @@ syscallfmt(uint8_t what, int syscallno, Ar0 *ar0, uint64_t start, uint64_t stop,
 		fmtprint(&fmt, "%d ", i[0]);
 		v = va_arg(list, void*);
 		l = va_arg(list, int32_t);
-		vl = va_arg(list, int64_t);
-		if (what == 'E') {
-			fmtprint(&fmt, "%#P %ld 0x%llx", v, l, vl);
+		fmtprint(&fmt, "%d %#p %ld", i[0], v, l);
+		if(syscallno == PREAD){
```

Comment:
This is already within a switch on syscallno, in the 'case PREAD' section

Reply:
yeah. The IF was only here back when the case was
case READ:
case PREAD: